feature-group-definitions/has.yml

```diff
@@ -1,4 +1,5 @@
 name: ":has()"
+description: The `:has()` CSS pseudo-class selects an element if any of the selectors passed as parameters would match at least one element.
```

Comment:
Can you move this into a separate PR together with `:is()` and `:where()`? I have some thoughts that I think should apply to all, and hopefully that'll be more effective done as one.

Reply:
See #862.